samples/Project GrandMaster/Assets/MixedRealityToolkit.Examples

```diff
@@ -0,0 +1 @@
+../../../external/MixedRealityToolkit-Unity/Assets/MixedRealityToolkit.Examples
```

Comment:
Also, if you aren't actually making use of the MRTK Examples, you don't have to add this symlink.

Reply:
The round pressable buttons on the board menu are from MRTK Examples. I'm not quite sure what else is used from that folder, from the top of my head, but if you'd like I can transfer those used prefabs to a 'resource' folder in the project's directory.

Reply:
Ah, then that's ok, let's keep it.